README.md

```diff
@@ -14,6 +14,8 @@ Once the dependencies are installed, run
 or, to build the full suite of utilities:
 
     make all
+    
+Note that when changing branches `make clean` may be required.
```

Comment:
or `make clean-geth` ?

Reply:
To answer your question I tested `make clean-geth` and it's insufficient.  This leads directly to the problem / solution.  Unlike for mobile builds, libbls_zexe.a source changes aren't tracked.  The library is never updated.  Marking libbls_exe.a PHONY in the Makefile fixes the behavior.